cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -39,7 +39,7 @@
 default["bcpc"]["hadoop"]["namenode"]["jmx"]["port"] = 10111
 default["bcpc"]["hadoop"]["namenode"]["rpc"]["port"] = 8020
 default["bcpc"]["hadoop"]["namenode"]["http"]["port"] = 50070 
-default["bcpc"]["hadoop"]["namenode"]["https"]["port"] = 50080 
```

Comment:
This was changed as per [`Hortonworks Documentation`](http://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.2.0/HDP_Man_Install_v22/index.html#Item1.26.3.4.3).